docs/network/host_networking.md

```diff
@@ -0,0 +1,346 @@
+
+## Table of Contents
+
+- [host ports](#host-ports)
+- [ips](#ips)
+- [interfaces](#interface)
+- [iptable](#iptable)
+  - [ovn-kubernetes](#ovn-kubernetes)
+  - [kubelet](#kubelet)
+- [firewalld](#firewalld)
+- [ip forward](#ip-forward)
+
+## Host Ports
+
+The following host ports are allocated for microshift components by default:
+
+|port    |namespace          |pod                |comment                                |
+|:-------|:------------------|:------------------|:--------------------------------------|
+|80      |openshift-ingress  |router-default-xxx |http                                   |
+|443     |openshift-ingress  |router-default-xxx |https                                  |
+
+Host ports exposure are implemented via iptable rules, see section [iptable -> kubelet](#kubelet) for details on how iptable rules are configured.
+
+## IPs
+
+Besides the cluster network subnet (default: 10.42.0.0/16) and cluster service subnet (default: 10.43.0.0/16), ovn-kubernetes uses the following IPs in its internal implementation:
+
+|ip                     |proto      | comment                                                          |
+|:----------------------|:----------|:-----------------------------------------------------------------|
+|169.254.169.1          |v4         |used for ipv4 hairpin traffic from host to hostservice            |
+|169.254.169.2          |v4         |used for ipv4 traffic from host to service                        |
+|169.254.169.3          |v4         |used for ipv4 nodeport service with externalTrafficPolicy=Local   |
+|169.254.169.4          |v4         |used as default gateway IP when ipv4 default route doesn't exist  |
+|100.64.0.0/16          |v4         |used in ovn logical switch                                        |
+|fd69::1                |v6         |used for ipv6 hairpin traffic from host to hostservice            |
+|fd69::2                |v6         |used for ipv6 traffic from host to service                        |
+|fd69::3                |v6         |used for ipv6 nodeport service with externalTrafficPolicy=Local   |
+|fd69::4                |v6         |used as default gateway IP when ipv6 default route doesn't exist  |
+|fd98::/64              |v6         |used in ovn logical switch                                        |
+
+**NOTE:** These IPs shall be reserved in user deployment.
+
+## Interfaces
+
+The following network interfaces are created or modified by ovn-kubernetes by default:
+
+|name               |type              |description        |comment                                                                                             |
+|:------------------|:-----------------|:------------------|:---------------------------------------------------------------------------------------------------|
+|br-ex              |OVS bridge        |gateway bridge     |created by microshift-ovs-init.service or manually (disableOVSInit = true)                          |
```

Comment:
should we also talk about br-ex1 ? may be another PR

Reply:
good catch, will fix it.

Reply:
added br-ex1.